library/CMakeLists.txt

```diff
@@ -174,14 +178,14 @@ endif(USE_STATIC_MBEDTLS_LIBRARY)
 if(USE_SHARED_MBEDTLS_LIBRARY)
 
     add_library(mbedx509 SHARED ${src_x509})
-    set_target_properties(mbedx509 PROPERTIES VERSION 2.20.0 SOVERSION 1)
+    set_target_properties(mbedx509 PROPERTIES VERSION 2.19.1 SOVERSION 1)
```

Comment:
Blocker: bad merge/rebase: this commit should not touch version numbers (here and below)

Reply:
Fixed.